### ISSUE TITLE
Borgs sometimes not recieving law 0.

### ISSUE THREAD

**Issue by ktccd** [tgstation/-tg-station#13337]
Problem Description: Cyborgs slaved to traitor AI's sometimes do not recieve the modified lawset, resulting in asimov borgs being ordered to do traitor things (Causing a lot of confusion)

What did you expect to happen: Normally borgs slaved to traitor ai's will have a law 0.

What happened instead: No law 0 was found (Though I think some of the description around the laws still say you have to help with his objectives?)

Why is this bad/What are the consequences: Because the immediate reaction by asimov borgs when ai tells them to kill X person is often to shout in common "AI IS MALF TRYING TO KILL X!"

Steps to reproduce the problem: I never did anything special except spawn as roundstart cyborg and the ai was a traitor.

Server: Sybil

Revision: 2015-11-30
0142846370eb1abf3bdb20b2ae51c6fe69cc1d9f 

Possibly related stuff (which gamemode was it? What were you doing at the time? Was anything else out of the ordinary happening?): As far as I could tell it was traitor, or maybe traitorlings.

**Comment by Anonus** [comment]
The last time this was reported it got called "not a bug" because "it's so that borgs don't start attacking people because lel traitor law time to griff" which is terrible reasoning.

**Comment by Niknakflak** [comment]
To be completely honest though, I can swear I've  seen traitor AIs spawn with borgs that have a law 0 and borgs that don't.

**Comment by GunHog** [comment]
A borg slaved to a Traitor AI is meant to get a law that tells it to follow its AI's objectives.

**Comment by Anonmare** [comment]
I assumed it was so that if some chuckle fuck immediately de-synced the borg and the AI never told them they had a Law 0 then they could use that ignorance to stop them from being valid'd instantly

**Comment by Anonus** [comment]
They CAN check laws to get the law, though, so it's a stopgap measure at best, plus the AI isn't going to not tell the borg it's rogue unless it's a very bad AI.

**Comment by optimumtact** [comment]
this is definitely a bug.

**Comment by ktccd** [comment]
I got more info that may help with tracing where the bug is from! ^^ Here's my notes:
cyborg law bug:
I very likely had traitor antag OFF (I usually do, I'm fairly certain I did when I got the bug too)
person 2 had all antags ON
I was mediborg
person 2 was janiborg

both were roundstart borgs
I had SOME of the message from the law 0, but not the law. That is to say, I had some of the descriptive text telling me to obey the ai and it hinted that the ai had a law 0, but I didn't have the law 0 itself.
